verifier/submission-verifier.js

```diff
@@ -0,0 +1,48 @@
+const ERROR = require("../constants/error-constants");
+const {toPascalCase} = require("../utility/string-util")
+
+function verifySubmissionAction(submissionId, action){ 
+
+    return new SubmissionActionVerifier(submissionId, action);
+}
+
+class SubmissionActionVerifier {
+    constructor(submissionId, action){
+        if(!submissionId) throw Error("submissionID is required!");
```

Comment:
Could you declare this error messages in the error-constant.js?

Reply:
I will